application/src/main/java/org/thingsboard/server/controller/ImageController.java

```diff
@@ -92,7 +94,8 @@ public class ImageController extends BaseController {
     @PreAuthorize("hasAnyAuthority('SYS_ADMIN', 'TENANT_ADMIN')")
     @PostMapping("/api/image")
     public TbResourceInfo uploadImage(@RequestPart MultipartFile file,
-                                      @RequestPart(required = false) String title) throws Exception {
+                                      @RequestPart(required = false) String title,
+                                      @RequestPart(required = false) Boolean isPublic) throws Exception {
```

Comment:
default "true"

Reply:
no `defaultValue` prop in `@RequestPart`